example/deployment/AWS-EMR/README.md

```diff
@@ -0,0 +1,127 @@
+    3. You will need to create a key pair which allows you to SSH into the EMR cluster and for Terraform to provision the cluster:
+        1. Create a key pair by logging in and from the AWS console, select Services -> EC2. Then select Key Pairs from the side menu.
+        2. Click "Create Key Pair" which will ask for a name and give you a private certificate to download.
+        
+       Put the name of your key pair in the `key_name` field  as it appears in AWS (no `.pem` extension).
+    
+    4. `pem_file` should be the path to the private key create in previous step (ending in `.pem` extension).
+    5. `vpc_id` is the ID of the Virtual Private Cloud the EMR cluster will connect to. From AWS console, select Services -> VPC, then choose Your VPCs from the side menu.
+    You may create a new VPC for this example or use an existing one. Place the name in the `vpc_id` field.
+    6. The `ingress_ip_range` is a list of public IPs that will be able to connect into the EMR cluster. This should be your public IP for your client machine.
+    7. The `subnet_id` should be the ID of a valid subnet from the Subnets page of the VPC service in AWS. It should be part of the VPC named earlier.
+    8. **Optional**: Change the AWS region by uncommenting the `aws_region` key and setting a region name.
+4. Run the script below to start creating instances on AWS and run the example. It requires the key pair PEM file from earlier. 
+
+    ```./example/deployment/AWS-EMR/bash-scripts/runAWS-EMRExample.sh <path to PEM private key file>```
+5. Answer `yes` when asked if Terraform may create AWS infrastructure.
+**You will start to incur charges to your AWS subscription at this point!**
+6. It will take several minutes to deploy the EMR cluster and run the Palisade example.
+7. Once complete, you can SSH into the master node by finding Services -> EMR from the AWS console, then selecting the name of the EMR
+    instance created by Palisade (something like "PalisadeExample") and then selecting the "SSH" link from the "Master Public DNS" section near the top.
+    Follow the instructions to log in. Ensure you are logged in as the "hadoop" user.
+8. Running the command below inside the SSH terminal should list the outputs from the MapReduce job:
+
+   ```hdfs dfs -ls output```
+   
+   which should look something like:
+   
+    ```
+    Found 2 items
+    -rw-r--r--   1 hadoop hadoop          0 2019-07-11 14:02 output/_SUCCESS
+    -rw-r--r--   1 hadoop hadoop         10 2019-07-11 14:02 output/part-r-00000
+    ```
+
+    Run the following to display the output file:
+    
+    ```hdfs dfs -text output/part-r-00000```
+    
+    which will produce output similar to this (you actual counts and values may differ):
+    ```bash
+    hdfs dfs -text output/part-r-00000
+    11500L     10
+    ```
```

Comment:
```suggestion
    ```

    This result comes from 6 queries of the data acting under different users and different purposes where only one query of the data returned the tax code.
```